app/scripts/templates/sign_in.mustache

```diff
@@ -72,7 +72,7 @@
         </div>
 
         <div class="button-row">
-          <button id="submit-btn" type="submit" class="disabled">{{buttonSignInText}}</button>
```

Comment:
This reminds me, with this experiment, [do we no longer care if errors are passed from one view to the next](https://github.com/mozilla/fxa-content-server/blob/78658f094d9862df24aa3f767ef40b24de5d6a61/app/scripts/views/form.js#L103:L105)?